### PR TITLE
Update useful commands reference in the README

### PR DESCRIPTION
**Pull request by mattseddon**
Small update to the (outdated) useful commands section of our README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,13 +127,12 @@ This extension augments VS Code in the following ways:
 Open the Command Palette (`F1` or ⇧⌃P on Windows/Linux or ⇧⌘P on macOS) and type
 in one of the following commands:
 
-| Command                    | Description                                                                                                                 |
-| -------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| `DVC: Get Started`         | Open the extension's walkthrough. Which details all of the current features and provides links to extra learning resources. |
```
Comment (mattseddon) [inline README.md:132]: [F] We want people to access this and `DVC: Setup The Workspace` via the setup page so I have removed both the entries.

File: README.md [hunk 2]
```diff
@@ -127,13 +127,12 @@ This extension augments VS Code in the following ways:
...
-| `View: Show DVC`           | Open the extension's view container.                                                                                        |
-| `DVC: Setup The Workspace` | Activate the extension's workspace setup wizard.                                                                            |
-| `DVC: Show Experiments`    | Show an interactive version of DVC's `exp show` [command](https://dvc.org/doc/command-reference/exp/show).                  |
-| `DVC: Show Plots`          | Show an interactive version of DVC's `plots diff` [command](https://dvc.org/doc/command-reference/plots/diff).              |
+| Command                 | Description                                                                                                    |
+| ----------------------- | -------------------------------------------------------------------------------------------------------------- |
+| `DVC: Show Setup`       | Show the extension's setup page. Which can be used to setup the workspace, view the walkthrough and more.      |
```
Comment (mattseddon) [inline README.md:132]: [Q] Any updates needed for this copy?

Comment (shcheklein) [reply README.md:132]: Which can be used to setup the DVC project?

Comment (mattseddon) [reply README.md:132]: ok

**Review by shcheklein — APPROVED** [review]

**Pull request merged**